zebra-rpc/src/methods/get_block_template_rpcs/types/unified_address.rs

```diff
@@ -0,0 +1,62 @@
+//! Types for unified addresses
+
+/// `z_listunifiedreceivers` response
+#[derive(serde::Serialize, serde::Deserialize)]
```

Comment:
Nitpick: this struct will be easier to work with if it implements common traits.

```suggestion
#[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
```

Reply:
I'm just going to commit this, it will make future tests easier.